app/src/main/java/com/github/se/eventradar/model/data/EventDatabase.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.se.eventradar.model.data
+
+import com.github.se.eventradar.model.repository.EventRepository
+import com.github.se.eventradar.model.repository.FirebaseEventRepository
+import com.github.se.eventradar.model.repository.MockEventRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class EventDatabase {
+  @Binds
+  abstract fun bindFirebaseEventRepository(userRepository: FirebaseEventRepository): EventRepository
+
+  @Binds abstract fun bindMockUserRepository(userRepository: MockEventRepository): EventRepository
```

Comment:
imp: I think there was a typo for this and maybe renaming the parameter to something other than userRepository could help avoid confusion, something like bindMockEventRepository(mockEventRepository: MockEventRepository)